frontend/containers/DashboardContainer.vue

```diff
@@ -21,6 +21,12 @@ export default Vue.extend({
     components: {
         DashboardAuctionsView,
     },
+    props: {
+        network: {
+            type: String,
+            default: undefined,
+        },
+    },
```

Comment:
You actually don't need to pass it as a prop, since it's already a container and can use `...mapGetters(network, { network: 'getMakerNetwork' })` directly